### ISSUE TITLE
Ectopic tooth eruption

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9565]
**Preferred term label:**
Ectopic tooth eruption

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
606893 
PMID: 27476657
8/8   

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also --  intraosseous vascular malformations  8/8 see article for definition

**Comment by pnrobinson** [comment]
added
